### DISCUSSION TITLE
Consider renaming this service

### DISCUSSION THREAD

**Discussion by cthoyt** [Steinbeck-Lab/cheminformatics-microservice#306] [General]
The fact that the service is all accessible through an API makes the "Python" part of "Cheminformatics Python Microservice" part misleading. Also, I am personally opposed to creating more things with three letter acronyms, so maybe you can come up with something more memorable (and also short). One bad suggestion: Chemicroservice

**Comment by Kohulan** [comment]
We could simply call it "Cheminformatics Microservice". Which is much straight forward in my opinion.

**Comment by Kohulan** [comment]
I would consider this as rather a discussion point and not an issue.

**Comment by vcnainala** [comment]
Microservice is now renamed to “Cheminformatics Microservice”.
